### ISSUE TITLE
BUG: ML Tooling can't handle models with no y values

### ISSUE THREAD

**Issue by andersbogsnes** [andersbogsnes/ml_tooling#263]
If we have a model that has no y values when training, ML-tooling will fail, as it always does a train_test_split which fails when y is None.

**To Reproduce**
1. Define a model where `y=None`
```
class RandomRankEstimator(BaseEstimator, ClassifierMixin):
    def fit(x, y=None):
          ... do modelstuff here...
    def predict(x):
        ...predict...

class CovRanker(ModelData):
       def get_training_data():
             return x, None
       def get_prediction_data(x):
            return x

model = CovRanker(RandomRankEstimator)
model.train_estimator()
```
**Expected behavior**
`.train_estimator` should not fail if no y value is supplied

**Additional context**

- [x] I have updated ML Tooling to the latest version
- [x] I have checked if this issue has been reported previously

**Comment by thomasfrederikhoeck** [comment]
Could be solved in #275

**Comment by andersbogsnes** [comment]
solved by #348
